File: solucoes/2008/nivel jr/fase 2/js/minicalc.js
var N, D, Q

scanf("%d%d%d", "N", 'D', "Q")

if(N > D && N > Q){
    prinf("%d%d", D, Q)
}
else if(N > D/10 && N > Q/10){
    if(Number.isInteger(D/10) && Number.isInteger(Q/10)){
        printf("%d%d\n", D/10, Q/10)
    }
    else{
        printf("%s", "IMPOSSIVEL")
    }
}

File: solucoes/2008/nivel jr/fase 2/py/minicalc.py
N, D, Q = list(map(int, input().split()))

if N > D and N > Q:
    print(int(D), int(Q))
elif((N > D/10) and (N > Q/10)):
    if(D/10).is_integer() and (Q/10).is_integer():
        print(int(D/10), int(Q/10))
    else:
        print('IMPOSSIVEL')